src/components/modules/document/DocRequiredCreation/DocRequiredCreation.container.vue

```diff
@@ -1,100 +1,122 @@
 <template>
   <doc-layout
     :is-closable="isClosable"
-    @close="handleCancel"
+    :is-returnable="isReturnable"
+    @return="onReturn"
+    @close="onCancel"
```

Comment:
`onClose`

Reply:
container does not close anything, container call cancelation by close event